modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -989,13 +990,25 @@ public void setAopProxy(Object aopProxy) {
 				_workflowStatusModelPreFilterContributor,
 				_userGroupRoleLocalService);
 
+		List<ObjectAction> objectActions =
+			_objectActionLocalService.getObjectActions(
```

Comment:
Please create a new finder using the `active` and `objectActionTriggerKey` fields, so that we can only return active actions of standalone type.

Reply:
Done ! The finder already exists.